hack/lib/util.sh

```diff
@@ -729,7 +729,7 @@ function kube::util::ensure-cfssl {
 # Check if we have "docker buildx" commands available
 #
 function kube::util::ensure-docker-buildx {
-  if docker buildx >/dev/null 2>&1; then
+  if docker buildx version >/dev/null 2>&1; then
```

Comment:
Is anything _else_ incompatible?

Reply:
Reworked to `if docker buildx version || docker buildx; then ...`. Either of them must succeed to declare `buildx` as supported.

Reply:
checking with `version` is fine, I mean that we've had explicit checks in other places at times that we're actually using docker because podman has a tendency to not actually be drop-in compatible with any of the commands, and we don't really have the bandwidth for that in the k8s builds

whereas this patch on it's own was fine.

Reply:
(https://github.com/kubernetes/kubernetes/issues/79048#issuecomment-502847403)

Reply:
> whereas this patch on it's own was fine.

+10000